### PR TITLE
Re-add runMonitoring

### LINKED ISSUE #2340: Option to monitor a slice
Re-add the option to monitor a slice of jobs with `runMonitoring`

### PR DESCRIPTION
**Pull request by mesmith75**
Fixes #2340

### CODE REVIEW

File: ganga/GangaCore/Core/MonitoringComponent/MonitoringService.py [hunk 1]
```diff
@@ -226,3 +226,28 @@ def stop(self):
...
+        Note:         
+          This method is meant to be used in Ganga scripts to request monitoring on demand. 
+        """
+
+        log.debug("runMonitoring")
+        if not self.alive:
+            log.error("Cannot run the monitoring loop. It has already been stopped")
+            return False
+
+        self.enabled = True
```
Comment (mesmith75) [inline ganga/GangaCore/Core/MonitoringComponent/MonitoringService.py:253]: I am not wholly enthused by this. You have to set `enabled` to `True` to monitor the jobs but in reality it is not monitoring everything.

**Review by egede — COMMENTED** [review]
What happens if I call `runMonitoring` when the monitoring is already running automatically. Will it do nothing, start a second parallel operation (and maybe cause trouble), do nothing, or make the whole thing crash?

**Comment by mesmith75** [comment]
It does nothing

**Comment by mesmith75** [comment]
So this currently monitors one job and then monitors all of them.

**Comment by mesmith75** [comment]
This is ready for review.

**Review by alexanderrichards — APPROVED** [review]

**Pull request merged**